website/videoBlog/2024-02-07-Building-an-Open-Source-Data-Lake-House-with-Hudip-Postgres-Hive-Metastore-Minio-and-StarRocks.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Building an Open Source Data Lake House with Hudi, Postgres Hive Metastore, Minio, and StarRocks"
+last_modified_at: 2024-02-21T16:54:38.964863-07:00
+authors:
+- name: Soumil Shah
+category: blog
+image: /assets/images/video_blogs/2024-02-07-Building-an-Open-Source-Data-Lake-House-with-Hudip-Postgres-Hive-Metastore-Minio-and-StarRocks.png
+navigate: "https://www.youtube.com/watch?v=TqnLhqWJ5Qo"
+tags:
+- guide
+- beginner
+- apache hudi
+- starrocks
+- postgreSQL
```

Comment:
- change to `postgresql`
- Also add a `postgres` tag wherever there is https://hudi.apache.org/videos/tags/postgresql/

Reply:
@nfarah86  Can you please add `postgres` tag also wherever there is a tag for `postgresql` - https://hudi.apache.org/videos/tags/postgresql/ ? I don't see that resolved.

Reply:
@nfarah86  I dont think this was resolved still. I added a commit to resolve and take this forward.